tests/test_biophysical/test_animal/test_cow_calf/test_beef_cow_calf_calculator.py

```
@@ -0,0 +1,802 @@
+            sex="female",
+            body_condition_score=5.0,
+            days_pregnant=None,
+            days_in_milk=None,
+            parity=2,
+            target_adg=0.0,
+            mud_condition="none",
+            temperature_c=20.0,
+            ne_diet_concentration=1.1,
+            process_based_phosphorus_requirement=0.0,
+        )
+        with pytest.raises(ValueError):
```

Ruff (0.15.17)

[warning] `pytest.raises(ValueError)` is too broad, set the `match` parameter or use a more specific exception

(PT011)